bin/version-include.sh

```diff
@@ -47,6 +47,17 @@ function populateValuesYAML() {
       cat "$USER_DIR/logging/user.env" | sed 's/^/      /' >> "$v4mValuesYAML"
     fi
   fi
+
+  # Encrypt passwords stored in V4M Helm Chart
+  if echo "$OSTYPE" | grep 'darwin' > /dev/null 2>&1; then
+    sed -i '' "s/GRAFANA_ADMIN_PASSWORD=.*/GRAFANA_ADMIN_PASSWORD=***/g" "$v4mValuesYAML"
+    sed -i '' "s/ES_ADMIN_PASSWD.*/ES_ADMIN_PASSWD=***/g" "$v4mValuesYAML"
+    sed -i '' "s/LOG_LOGADM_PASSWD.*/LOG_LOGADM_PASSWD=***/g" "$v4mValuesYAML"
+  else
+    sed -i "s/GRAFANA_ADMIN_PASSWORD=.*/GRAFANA_ADMIN_PASSWORD=***/g" "$v4mValuesYAML"
+    sed -i "s/ES_ADMIN_PASSWD.*/ES_ADMIN_PASSWD=***/g" "$v4mValuesYAML"
+    sed -i "s/LOG_LOGADM_PASSWD.*/LOG_LOGADM_PASSWD=***/g" "$v4mValuesYAML"
```

Comment:
These two lines (57 and 59) work but don't match link 57 since they do not included the "=" in the 2nd arg to sed.  Just to improve readability, let's make them all match and add the "=" to the other two lines.

Reply:
Good catch!  Updated the sed commands to include the "=" sign.